publisher-sdk/src/main/java/com/criteo/publisher/BidManager.java

```diff
@@ -245,18 +246,23 @@ private void sendBidRequest(List<CacheAdUnit> prefetchCacheAdUnits) {
   }
 
   void setCacheAdUnits(@NonNull List<CdbResponseSlot> slots) {
+    List<CdbResponseSlot> cachedCdbResponseSlots = new ArrayList<>();
```

Comment:
What is this list used for ?

Reply:
Oops, left over from an initial direction that I gave up on later. REmoved.